### PR TITLE
Windows support etc

### PR DESCRIPTION
**Pull request by pllim**
Better Windows support. Make instructions clearer. Added missing packages. More verbose check_env. Other minor edits.

Was originally sosey/aas229_workshop#2 but that did not go into #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,44 @@
 Using Python for Astronomical Data Analysis in the Era of JWST
 ================================================================
+
 The Space Telescope Science Institute and core developers from the Astropy community are sponsoring a workshop at the January 2017 meeting of the [American Astronomical Society](http://aas.org/) #AAS229 in Grapevine, Texas.
 
-Check out the [Schedule!](Schedule.md)
+Check out the [Schedule](Schedule.md)!
```
Comment (sosey) [inline README.md:7]: why did you move the bang? I was using this example: https://help.github.com/articles/relative-links-in-readmes/

**Review by sosey — APPROVED** [review]
seems fine, move the bang back for consistency and you can merge.

File: README.md (hunk 1 above)
Comment (pllim) [reply README.md:7]: Ops, sorry. I thought it was a typo. I changed it back.
